src/read.c

```diff
@@ -1968,52 +1968,36 @@ static avifBool avifParseToneMappedImageBox(avifGainMapMetadata * metadata, cons
         return AVIF_FALSE;
     }
 
+    uint16_t minimumVersion;
+    AVIF_CHECK(avifROStreamReadU16(&s, &minimumVersion));
```

Comment:
If possible it would be good to keep original names verbatim, for example:

```c
AVIF_CHECK(avifROStreamReadU16(&s, &minimumVersion)); // unsigned int(16) minimum_version;
```

It makes it easier to find something from the spec. Also for consistency with the rest of the file.

Reply:
Done.